assets/components/paymentMethods/paymentMethods.jsx

```diff
@@ -19,6 +18,8 @@ type PropTypes = {
   lastName: string,
   error: ?string,
   payPalButtonExists: boolean,
+  stripeCallback: Function,
+  paypalCallback: Function,
 };
```

Comment:
After an offline discussion, we realised that we need to be hooked to the redux state here because we want to hide the payment's buttons when the user interact with the form.